app/bundles/SmsBundle/Config/config.php

```diff
@@ -100,19 +100,25 @@
                 'alias' => 'sms_api',
             ],
             'mautic.sms.transport_chain' => [
-                'class'     => '\Mautic\SmsBundle\Sms\TransportChain',
-                'arguments' => ['%mautic.sms_transport%', 'mautic.helper.integration', 'monolog.logger.mautic'],
+                'class'     => \Mautic\SmsBundle\Sms\TransportChain::class,
```

Comment:
why is this? i usually don't do it, i like to keep configuration static. i believe it should be text.

Reply:
We this this format almost everywhere. Only the old code is written with strings - `::class` is used when Mautic dropped support of old PHP.